guide/guide.md

```diff
@@ -101,6 +101,45 @@ Do not put any spaces around the `.`.
 x.foo().bar().baz(x, y, z);
 ```
 
+### Match
+
+For a single-expression matcher to a single-expression matchee, which fits on
```

Comment:
Mayber "matcher" and "matchee" could simply be referred to as left-hand and right-hand side of a match arm?